profiles/arch/loong/package.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 2022-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Holger Hoffstätte <holger@applied-asynchrony.com> (2025-01-09)
```

Comment:
Please drop the ~loong keyword instead of masking: https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades.

> Note that it is preferred to drop keywords on the package and request rekeywording of it together with its new dependencies within the same bug to allow the new code path(s) in your package to be tested. This won't happen if the new dependency is requested for keywording by itself and package.use.mask is used to mask the relevant new USE flag: only the new package dependency will be tested by arch testers. Also, the mask has to be manually removed during the testing process, which is cumbersome.

Reply:
OK, I'll drop loong from 4.4.3 & 9999.